i18n/zh-CN/docusaurus-plugin-content-docs/current/guides/libraries.md

```diff
@@ -21,6 +22,19 @@ FastLED
 MySensors@2.3.0
 ```
 
-You can find the library names in the Arduino Library Manager. Alternatively, you can find it inside the source code of the library. Look for the `name` field of the `library.properties` file.
+You can find the library names in the Arduino Library Manager. Alternatively, you can find it inside the source code of the library. Look for the `name` field of the `library.properties` file.（这句话不太懂什么意思？？）
```

Comment:
You wrote some question in Chinese, I guess it is not part of the translation. How can I help?